templates/kobo-env/enketo_express/config.json.tpl

```diff
@@ -7,6 +7,7 @@
     },
     "encryption key": "${ENKETO_ENCRYPTION_KEY}",
     "less secure encryption key": "${ENKETO_LESS_SECURE_ENCRYPTION_KEY}",
+    "support": "{DEFAULT_FROM_EMAIL}",
```

Comment:
@rroux . I think the `$` is missing as in "${DEFAULT_FROM_EMAIL}",

Reply:
I just added the $